sdk/sdk-common/src/common/implementation/TokenAmount.ts

```diff
@@ -83,6 +83,14 @@ export class TokenAmount implements ITokenAmount {
     return new BigNumber(this.amount).times(this._baseUnitFactor)
   }
 
+  public fromBaseUnit(): string {
```

Comment:
I'm not sure at all about these: the value stored in TokenAmount should have only one meaning/format. Either is in base unit, or it is a floating point string representation, but with these methods it feels like you need to know what that representation is, and there is no way of knowing unless you inspect the value. What is the purpose of these methods?

Reply:
Removed. Was an error actually.